iac-extensions/arm/src/test/resources/checks/LogRetentionCheck/Microsoft.Network_networkWatchers_flowLogs.bicep

```diff
@@ -0,0 +1,125 @@
+resource flowLogs1 'Microsoft.Network/networkWatchers/flowLogs@2022-07-01' = {
+  name: 'Noncompliant: Raise Issue as days is too low /'
+  properties: {
+    storageId: 'dummy'
+    targetResourceId: 'dummy'
```

Comment:
Nitpick: if those elements are not needed for the rule, we could remove them.
Not a big deal, we can also keep them if you think they make sense.

Reply:
Good question. I keep them because Bicep builder `az bicep build` throws errors when they are missing (those fields are mandatory in the schema). The same with `/` in `name` field.
I will remove them to simplify test cases.